src/prefabs/icon.js

```diff
@@ -1278,12 +1278,180 @@
+          key: 'addBadge',
+          value: false,
+        },
+        {
+          type: 'VARIABLE',
+          label: 'Content',
+          key: 'content',
+          value: ['1'],
+          configuration: {
+            as: 'MULTILINE',
+            condition: {
+              type: 'SHOW',
+              option: 'addBadge',
+              comparator: 'EQ',
+              value: true,
+            },
+          },
+        },
+        {
+          label: 'Badge Color',
+          key: 'badgeColor',
+          value: 'default',
+          type: 'CUSTOM',
+          configuration: {
+            as: 'BUTTONGROUP',
+            dataType: 'string',
+            allowedInput: [
+              { name: 'Default', value: 'default' },
+              { name: 'Error', value: 'error' },
+              { name: 'Primary', value: 'primary' },
+              { name: 'Secondary', value: 'secondary' },
+            ],
+            condition: {
+              type: 'SHOW',
+              option: 'addBadge',
+              comparator: 'EQ',
+              value: true,
+            },
+          },
+        },
+        {
+          label: 'Anchor Origin',
+          key: 'anchorOrigin',
+          value: 'right,top',
+          type: 'CUSTOM',
+          configuration: {
+            as: 'BUTTONGROUP',
+            dataType: 'string',
+            allowedInput: [
+              {
+                name: 'Top Right',
+                value: 'right,top',
+              },
+              {
+                name: 'Top Left',
+                value: 'left,top',
+              },
+              {
+                name: 'Bottom Right',
+                value: 'right,bottom',
+              },
+              {
+                name: 'Bottom Left',
+                value: 'left,bottom',
+              },
+            ],
+            condition: {
+              type: 'SHOW',
+              option: 'addBadge',
+              comparator: 'EQ',
+              value: true,
+            },
+          },
+        },
+        {
+          label: 'Variant',
+          key: 'variant',
+          value: 'standard',
+          type: 'CUSTOM',
+          configuration: {
+            as: 'BUTTONGROUP',
+            dataType: 'string',
+            allowedInput: [
+              { name: 'Standard', value: 'standard' },
+              { name: 'Dot', value: 'dot' },
+            ],
+            condition: {
+              type: 'SHOW',
+              option: 'addBadge',
+              comparator: 'EQ',
+              value: true,
+            },
+          },
+        },
+        {
+          type: 'SIZES',
+          label: 'Outer Space',
+          key: 'margin',
+          value: ['S', 'S', 'S', 'S'],
+          configuration: {
+            condition: {
+              type: 'SHOW',
+              option: 'addBadge',
+              comparator: 'EQ',
+              value: true,
+            },
+          },
+        },
+        {
+          type: 'TOGGLE',
```

Comment:
Why do we need this option? First define a badge, then decide to not show it? Is this for toggling the visibility maybe? Then I would expect a `B.defineFunction` which you can use in an interaction.
And please when this option is really needed, use the Key and label `visibility` to avoid the double negative. Now it's `invisible === false`.
Then in your component you can do `invisible="!visible"`

Reply:
Removed.